### DISCUSSION TITLE
Film flow validation

### DISCUSSION THREAD

**Discussion by lecorrejm** [OpenSTREAM-solvers/openstream#23] [Validations]
Use the data from Adamsson and Anglart which include the axial distribution of steady-state film flow for various axial power distributions.
https://www.sciencedirect.com/science/article/pii/S0029549306001701

**Comment by jasonjan2003** [comment]
The Adamsson2006 dataset have been included in [mfval/TwoPhase-Database](https://github.com/mfval/TwoPhase-Database). 
Further discussion should take place in that repository.
